tests/precommit_test.py

```diff
@@ -131,22 +134,35 @@ def test_get_added_and_existing_findings_in_changes_for_changes(self):
         self.assert_findings_ids(self.precommit_client.findings_in_changed_code, [])
         self.assert_findings_ids(self.precommit_client.existing_findings, [4, 5, 6, 7])
 
+    @responses.activate
+    def test_adding_path_prefix(self):
+        path_prefix = 'prefix/'
+        self.precommit_client = self._get_precommit_client(self._get_changed_file(),
+                                                           self._get_no_deleted_files(), path_prefix=path_prefix)
```

Comment:
I read that `_get_no_deleted_files()` implies we do not test the transformation of "deleted" paths. I would prefer if we did.